bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI/Controllers/SystemUserController.cs

```diff
@@ -47,8 +47,8 @@ public SystemUserController(ISystemUserService systemUserService, IHttpContextAc
     public async Task<ActionResult> GetSystemUserListForLoggedInUser(CancellationToken cancellationToken = default)
     {
         var (partyId, actionResult) = ResolvePartyId();
-        
-        if (partyId == 0) return actionResult;
+
+        if (partyId == 0) return actionResult;               
```

Comment:
what will be the action result here when the party is 0?

Reply:
and what is the check here? to verify if the reportee is an organisation?

Reply:
BadRequest("PartyId not provided in the context.") or 500 server error. It returns the second value from the tuple result from the ResolvePartyId method above.